integration_tests/pytest.ini

```diff
@@ -26,5 +26,8 @@ markers =
     validate_execs_in_gpu_plan([execs]): Exec class names to validate they exist in the GPU plan.
     shuffle_test: Mark to include test in the RAPIDS Shuffle Manager
     premerge_ci_1: Mark test that will run in first k8s pod in case of parallel build premerge job
+    nightly_test_to_cases: tests either time-consuming or mem-consuming, we split them into cases in nightly
```

Comment:
I'm not sure I understand what the to_cases description here?  does this mark entire file and then if you want just a single test you use the nightly_*case ones?

Reply:
nightly_test_to_cases this mark the entire test file (most of cases are time consuming) which will be split into cases in parallel run

So we parallel
1. special cases in nightly_test_to_cases
2. nightly_test_to_cases
3. other tests files

let me try rename this to something meaningful

Reply:
@tgravescs renamed to `nightly_resource_consuming_test`, please let me know if this sounds OK, thx~